### PR TITLE
:fire: mets server did not support local_filename and url

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

**Comment by MehmedGIT** [comment]
Looks good

**Review by MehmedGIT — APPROVED** [review]
Looks good.

File: tests/test_mets_server.py [hunk 1]
```diff
@@ -32,11 +32,11 @@ def _start_mets_server(*args, **kwargs):
     if exists(WORKSPACE_DIR):
         rmtree(WORKSPACE_DIR, ignore_errors=True)
 
-    copytree(assets.path_to('kant_aufklaerung_1784/data'), WORKSPACE_DIR)
+    copytree(assets.path_to('SBB0000F29300010000/data'), WORKSPACE_DIR)
     workspace = Workspace(Resolver(), WORKSPACE_DIR)
     p = Process(target=_start_mets_server, kwargs={'workspace': workspace, 'url': request.param})
     p.start()
-    sleep(2)  # sleep to start up server
+    sleep(.1)  # sleep to start up server
```
Comment (MehmedGIT) [inline tests/test_mets_server.py:39]: I would still keep this high enough - 1 second.

**Review by MehmedGIT — CHANGES_REQUESTED** [review]
Since the python 3.7 test failed, I think this is because of the low sleep in the test.

File: tests/test_mets_server.py (hunk 1 above)
Comment (kba) [reply tests/test_mets_server.py]: Indeed, that was just for testing, corrected in 97a05facf

**Pull request merged**